models/db/error.go

```diff
@@ -42,3 +42,18 @@ func IsErrSSHDisabled(err error) bool {
 func (err ErrSSHDisabled) Error() string {
 	return "SSH is disabled"
 }
+
+// ErrNotExist represents a non-exist error.
+type ErrNotExist struct {
+	ID int64
```

Comment:
If it is that general, perhaps we should add a `Type string` as well:
```suggestion
	ID int64
	Type string
```

Reply:
I personally disagree with extending it,

because it's used only in Get_BeanXY_By_Z()  and so the type is always clear

Reply:
I think we should not have this error. It's too common to prevent we know which error it is.